package-lock.json

```diff
@@ -1,8 +1,8692 @@
 {
   "name": "bpmn-js-executable-fix",
   "version": "0.1.3",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Please keep `lockfileVersion: 1`, as our downstream libraries and CI use old LTS Node versions

Reply:
hm, any ideas how I can do that on the latest NPM version?

Reply:
I don't think it is possible with the latest version. I'm using nvm to switch to `node 12`. If it is okay with you, I can also bump the versions and close your PRs